vocode/streaming/synthesizer/polly_synthesizer.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+from concurrent.futures import ThreadPoolExecutor
+import logging
+from typing import Any, Optional
+import aiohttp
+
+from vocode.streaming.agent.bot_sentiment_analyser import BotSentiment
+from vocode.streaming.models.message import BaseMessage
+from vocode.streaming.synthesizer.base_synthesizer import (
+    BaseSynthesizer,
+    SynthesisResult,
+    tracer,
+)
+from vocode.streaming.models.synthesizer import PollySynthesizerConfig, SynthesizerType
+from vocode.streaming.utils.mp3_helper import decode_mp3
+
+import boto3
+
+class PollySynthesizer(BaseSynthesizer[PollySynthesizerConfig]):
+    def __init__(
+        self,
+        synthesizer_config: PollySynthesizerConfig,
+        logger: Optional[logging.Logger] = None,
+        aiohttp_session: Optional[aiohttp.ClientSession] = None,
+    ):
+        super().__init__(synthesizer_config, aiohttp_session)
+
+        client = boto3.client("polly")
+
+        self.client = client
+        self.language_code = synthesizer_config.language_code
+        self.voice_id = synthesizer_config.voice_id
+        self.thread_pool_executor = ThreadPoolExecutor(max_workers=1)
+
+    def synthesize(self, message: str) -> Any:
+        # Perform the text-to-speech request on the text input with the selected
+        # voice parameters and audio file type
+        return self.client.synthesize_speech(
+            Text=message, 
+            LanguageCode=self.language_code,
+            TextType="text", 
+            OutputFormat="mp3",
```

Comment:
looks like polly supports `pcm`, can we switch to that? it's easy enough to slap the wav header on that - makes this much more efficient!

Reply:
we also have the sampling rate that we want in the `synthesizer_config` which can also be passed in here

Reply:
done, one tidbit here is polly only support 8k and 16k sample rate for pcm